src/Layout/ComputationMenu/ComputationMenu.tsx

```diff
@@ -334,9 +334,10 @@ export class ComputationMenu extends React.Component<ComputationMenuProps, {}> {
                         onYearChange={this.onYearChange}
                         year={this.props.settingsPayload.year}
                         tooltip={
-                            <TooltipInfoRight
+                            <TooltipInfo
                                 text={"Her kan du velge året stortingsvalget ble holdt."}
                                 url={WIKIURL + "#Valgt%20%C3%A5r"}
```

Comment:
This should probably have been encoded using a function rather than be done manually, but doesn't *need* fixing.